recipes/tilt/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set version = "0.16.1" %}
+{% set commit = "1f21e296d6a1029223d273bcf43dc6b0bf71c0e8" %}
+{% set date = "2020-07-17" %}
+
+package:
+  name: tilt
+  version: {{ version }}
+
+source:
+  url: https://github.com/tilt-dev/tilt/archive/v{{ version }}.tar.gz
+  sha256: 4df87f9efaa83e518d45f0ea10b6b3f7839076ab70731e434ce2f45f794770d4
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('go') }} 1.13
```

Do you need to specify a version? It should choose the newest by default:
```suggestion
    - {{ compiler('go') }}
```